tests/tools/onert_train/src/args.cc

```diff
@@ -199,6 +229,27 @@ void Args::Initialize(void)
     }
   };
 
+  // supported loss list and it's default value
+  static std::vector<NNFW_TRAIN_LOSS> valid_loss = {
```

Comment:
Can I ask why you declare `static` on it?

Reply:
AFAIK, `const` looks much better.

Reply:
I added `const`, here. 

> Can I ask why you declare static on it? 

`std::vector<NNFW_TRAIN_*>` reference is captured by the lambda. 
Since the lambda invoked when `Args::Parse` called, `std::vector<NNFW_TRAIN_*>` have to alive after `Args::Initialized()` is called. 
That's why I added static here.

Reply:
Hmm. I rethink this part. 
Since the `static` variable is alive throughout the whole program's lifetime. 
Instead of using `static`, I moved it into a class member.